contracts/characters.sol

```diff
@@ -116,6 +121,10 @@ contract Characters is Initializable, ERC721Upgradeable, AccessControlUpgradeabl
     mapping(uint256 => mapping(uint256 => uint256)) public nftVars;//KEYS: NFTID, VARID
     uint256 public constant NFTVAR_BUSY = 1; // value bitflags: 1 (pvp) | 2 (raid) | 4 (TBD)..
 
+    Garrison public garrison;
+    mapping(uint256 => bool) public isStaminaLocked;
+    mapping(uint256 => uint8) public lockedStaminaPoints;
```

Comment:
This will start failing if we max stamina exceeds 255, perhaps we should make it uint16 just to be safe otherwise this will become a zombie state var in the future